tests/functional/docs/test_duplicate_docs_block.py

```diff
@@ -1,34 +0,0 @@
-from dbt.tests.util import run_dbt
```

Comment:
So doesn't look like any adapter implements these tests and they're really covering dbt-core functionality. 

We will test this via the persist_docs tests defined in dbt-tests-adapter in https://github.com/dbt-labs/dbt-postgres/issues/4